models/stg_youtube.yml

```diff
@@ -68,30 +67,94 @@ models:
       - name: views_percentage
         description: Total percent of views the user makes up for the video
 
-  - name: stg_youtube__video_metadata
+  - name: stg_youtube__video
     description: >
-      Table containing individual video metadata. This table IS NOT synced by Fivetran. However, you may easily configure a Fivetran Cloud Function
-      as shown in our [blog post](tbd) to sync your Youtube video metadata to a table in your warehouse and have this dbt package generate the video
-      metadata table in this same schema which can then be used in downstream models.
+      Table containing individual video metadata such as the title, channel, description, thumbnails, and other video information.
     columns:
+      - name: _fivetran_synced
+        description: Timestamp of when the record was synced by Fivetran
+      - name: content_details_caption
+        description: Boolean indicating if the video contains captions
+      - name: content_details_definition
+        description: Identifies the details of the available definition of the video (ie. hd, sd, etc.)
+      - name: content_details_dimension
+        description: Identifies the dimension of the video (ie. 2d)
+      - name: content_details_duration
+        description: Identifies the duration of the video
+      - name: content_details_has_custom_thumbnail
+        description: Boolean indicating if the video has a custom thumbnail
+      - name: content_details_licensed_content
+        description: Boolean indicating if the video has licensed content
+      - name: content_details_projection
+        description: Identifies the projection type of the video
+      - name: content_details_region_restriction
+        description: Identifies the region restrictions of the video
+      - name: etag
+        description: Electronic tag of the video
       - name: video_id
```

Comment:
would it make sense to move the grain field to the topmost column for easier readability.

Reply:
Makes sense to bring it to the top for easier readability. Just updated. Thanks!